src/style.css

```
@@ -0,0 +1,34 @@
+body {
+  margin: 0;
+}
+
+.container {
+  height: 96px;
+  width: 96px;
+  background-color: #fff;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
```

Remove redundant properties
```suggestion
.stars {
  display: flex;
  align-items: center;
}
```